repo/meta/index.json

```diff
@@ -1,6 +1,9 @@
 {
   "packages":[
     {
```

Comment:
Why do we need this addition to index.json? This could be what is breaking the build.